perception/tracking_object_merger/CMakeLists.txt

```diff
@@ -0,0 +1,50 @@
+cmake_minimum_required(VERSION 3.8)
+project(tracking_object_merger VERSION 1.0.0)
+
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic -Werror -Wconversion)
```

Comment:
[imo]
Could you please correct the spelling errors here and in some of the code?

Reply:
Thanks. I fixed this comment in https://github.com/autowarefoundation/autoware.universe/pull/4340/commits/ff4b3c89992c7b6bc575d6fb80116ecbf880d3d0. (BTW `-Wconversion` is not spelling miss.)

Reply:
@YoshiRi 
Thanks! Even if it's not miss, we should not leave the error as it is.
Maybe adding comment like `# cspell:ignore SOMEWORD` is good way to resolve this.